recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -22,10 +22,24 @@ if "%CONDA_BLD_PATH%" == "" (
 )
 
 :: Increase pagefile size, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
-if "%CI%" == "azure" (
-    if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
-        REM this file will be run within %SCRIPTS%, where SetPageFileSize.ps1 is also copied into, see meta.yaml
-        call SetPageFileSize.ps1 -MinimumSize 4GB -MaximumSize 8GB -DiskRoot "D:"
+:: Both in the recipe and in the final package, this script is co-located with SetPageFileSize.ps1, see meta.yaml
+set ThisScriptsDirectory=%~dp0
+set EntryPointPath=%ThisScriptsDirectory%SetPageFileSize.ps1
+:: Only run if SET_PAGEFILE is set; EntryPointPath needs to be set before if not using delayed execution.
+if "%SET_PAGEFILE%" NEQ "" (
```

Comment:
Reorder the lines to scope environment variables and simplify comment (the "if" is self-explanatory, I don't understand the comment about EntryPointPath so I left it untouched: what is delayed excecution?)
```suggestion
if "%SET_PAGEFILE%" NEQ "" (
    :: Both in the recipe and in the final package, this script is co-located with 
    :: SetPageFileSize.ps1, see meta.yaml
    set ThisScriptsDirectory=%~dp0
    set EntryPointPath=%ThisScriptsDirectory%SetPageFileSize.ps1
    :: EntryPointPath needs to be set before if not using delayed execution.
```

Reply:
If the variables are set within the if-condition, they are not available for use within the script (yeah, windows...), unless delayed ~execution~ expansion is used: https://ss64.com/nt/delayedexpansion.html

That's potentially a larger change though that I didn't want to touch, and setting the variables before the if-condition doesn't hurt anyone, IMO.

Reply:
Ahh, ok. Never mind then.